index.html

```diff
@@ -138,7 +138,7 @@
                   brush
                   if-no-vr-headset="visible: false"
```

Comment:
I would probably change that logic. Hide the controllers by default and display when entering VR.

We might not need a component for just that and incorporate the logic in `paint-controls`:

```
// in init method.
this.onEnterVR = this.onEnterVR.bind(this);
this.el.sceneEl.addEventListener('enter-vr', this.onEnterVR);

// component method
onEnterVR: function () {
  this.el.object3D.visible = true;
}
```